test/runtests.jl

```diff
@@ -193,7 +193,9 @@ end
         box_size=box_size,
         neighbour_finder=DistanceNeighbourFinder(nb_matrix, 10),
         thermostat=AndersenThermostat(10.0),
-        loggers=Dict("temp" => TemperatureLogger(10)),
+        loggers=Dict("temp" => TemperatureLogger(10),
+                        "coords" => CoordinateLogger(10),
+                        "energy" => EnergyLogger(10)),
```

Comment:
Should we also add some energy conservation tests?
I'm not sure how well does it conserve in more complex cases.

Reply:
Probably longer term, though protein cases usually use a thermostat so energy won't conserve (though temperature should).